source-map-support.js

```diff
@@ -80,13 +80,17 @@ retrieveFileHandlers.push(function(path) {
 
   var contents = null;
   if (!fs) {
-    // Use SJAX if we are in the browser
-    var xhr = new XMLHttpRequest();
-    xhr.open('GET', path, false);
-    xhr.send(null);
-    var contents = null
-    if (xhr.readyState === 4 && xhr.status === 200) {
-      contents = xhr.responseText
+    try {
+      // Use SJAX if we are in the browser
+      var xhr = new XMLHttpRequest();
+      xhr.open('GET', path, false);
```

Comment:
Please add a comment like
`// Note: synchronous XHR`
It's very weird so a comment would alert readers that it is intended. 

or you could use `/** async */ false` in the call.